transformers4rec/torch/trainer.py

```diff
@@ -459,7 +461,8 @@ def evaluation_loop(
                 inputs,
                 prediction_loss_only,
                 ignore_keys=ignore_keys,
-                ignore_masking=ignore_masking,
+                training=False,
+                testing=True,
```

Comment:
the testing flag should be set based on the value defined in the `if description=="prediction":` statement. 
```suggestion
                testing=testing,
```